metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "simp-clamav",
-  "version": "6.0.1",
+  "version": "6.1.0",
```

Comment:
Honestly, this is a bug fix. We never should have forced users into only our choices for updates.

Reply:
@trevor-vaughan does that mean you think it should be 6.0.2 instead of 6.1.0?

Reply:
@op-ct Yes

Reply:
@trevor-vaughan :+1: updated